liblangutil/SourceReferenceFormatterHuman.cpp

```diff
@@ -105,7 +105,7 @@ void SourceReferenceFormatterHuman::printSourceLocation(SourceReference const& _
 
 	if (!_ref.multiline)
 	{
-		auto const locationLength = static_cast<size_t>(_ref.endColumn - _ref.startColumn);
+		size_t const locationLength = static_cast<size_t>(_ref.endColumn - _ref.startColumn);
```

Comment:
Why? It's already clear what the type is

Reply:
It is another indirection away.